.pipelines/pipeline.yaml

```diff
@@ -420,6 +420,18 @@ stages:
         k8sVersion: ""
         dependsOn: "test"
 
+    # Cilium Dualstack Overlay E2E tests
+    - template: singletenancy/cilium-dualstack-overlay/cilium-dualstackoverlay-e2e-job-template.yaml
+      parameters:
+        name: "cilium_dualstackoverlay_e2e"
+        displayName: Cilium on AKS DualStack Overlay
+        os: linux
+        clusterType: cilium-dualstack-up
```

Comment:
the cluster type here should be dualstack-byocni-nokubeproxy-up, cilium-dualstack-up is the target that will create a managed cluster, but for the pipelines we use the byocni option